docs/getting-started/installing-solidus.mdx

```diff
@@ -54,15 +53,19 @@ you a complete store. This is the approach we'll be following in this guide.
 
 ### In a new app
 
-If you don't have an existing Ruby on Rails application yet, simply create one with Sqlite3 as your
-database:
+If you don't have an existing Ruby on Rails application yet, simply create one with:
 
 ```bash
-$ rails new amazing_store
+$ rails new -T amazing_store
```

Comment:
nit: Using the expanded option `--skip-test` might be clearer for anyone unfamiliar with `rails new` options.

Reply:
I really wanted to keep the command as short as possible. It's already explained that we are skipping tests below but what I'm more explicit there about the `-T` option?